src/main/msp/msp.c

```diff
@@ -1502,17 +1502,19 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, rxConfig()->fpvCamAngleDegrees);
         sbufWriteU8(dst, 0); // not required in API 1.44, was rxConfig()->rcSmoothingChannels
 #if defined(USE_RC_SMOOTHING_FILTER)
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_mode);
+        sbufWriteU8(dst, 0); // not required in API 1.44, was rxConfig()->rc_smoothing_type
         sbufWriteU8(dst, rxConfig()->rc_smoothing_setpoint_cutoff);
         sbufWriteU8(dst, rxConfig()->rc_smoothing_feedforward_cutoff);
         sbufWriteU8(dst, 0); // not required in API 1.44, was rxConfig()->rc_smoothing_input_type
         sbufWriteU8(dst, 0); // not required in API 1.44, was rxConfig()->rc_smoothing_derivative_type
+        sbufWriteU8(dst, rxConfig()->rc_smoothing_mode);
```

Comment:
This has to be added to the end of the frame and needs a comment to let us know in which api version it was added.

Reply:
Not sure exactly how to do that.

Reply:
@klutvott123 thanks for your help - hopefully the end result is correct now?